optuna/samplers/_cmaes.py

```diff
@@ -23,6 +23,8 @@
 _logger = logging.get_logger(__name__)
 
 _EPS = 1e-10
+# The value of system_attrs must be less than 2046 characters on RDBStorage.
+_SYSTEM_ATTR_MAX_LENGTH = 2045
```

Comment:
Are there special characters involved? In the schema definition and the error messages say 2048.

Reply:
The value is double-quoted by `json.dumps()`. So `abc` is converted to `"abc"`.
https://github.com/optuna/optuna/pull/1833#discussion_r488526833

I think we can also set `_SYSTEM_ATTR_MAX_LENGTH = 2046`, but I put a margin of 1 character.